runall.sh

```diff
@@ -37,6 +37,11 @@ echo "Using compose file: ${COMPOSE_FILE}"
 
 [[ -z ${HSDS_ENDPOINT} ]] && echo "HSDS_ENDPOINT is not set" && exit 1
 
+# Do not allow credentials to expire by default.
+if [[ -z ${AUTH_EXPIRATION} ]] ; then
```

Comment:
You shouldn't need to set the default here - it will get picked up from the default in config.py if there's no environment variable set.

Reply:
This is actually one thing I did not understand. 

In `config.get()` it looks there's an asymmetry where, for example, `config.get('A')`  will give a return value if `A` in the environment variable, otherwise it will return `None`. On the otherhand `config.get('a')` will return the default value for `a` instead of `None`. It wasn't clear to me if this was intentional or a bug.

Either way, I'll remove this and update.